python/qpid_dispatch/management/client.py

```diff
@@ -76,6 +76,16 @@ def delete(self):
 class Node(object):
     """Client proxy for an AMQP management node"""
 
+    def clean_attrs(self, attrs):
+        BOOL_VALUES = {"yes": 1, "true": 1, "on": 1, "no": 0, "false": 0,
```

Comment:
What about case?  Is 'Yes' true or only 'yes'?

Reply:
This is tough question. I think 'Yes' should be invalid, so should 'YES' or 'YEs' or 'NO' or 'No'
Please let me know what your thoughts on that are.